interface/main/messages/messages.php

```diff
@@ -334,6 +334,7 @@
 
 if ($noteid) {
     $body = preg_replace('/(:\d{2}\s\()'.$result['pid'].'(\sto\s)/','${1}'.$patientname.'${2}',$body);
+    $body = preg_replace('/\sto\s/i',' '.xl('to').' ',$body);
```

Comment:
Actually prob best to make a new preg_replace that changes 'to' when:
begin_line . datestamp . word . "to" . word